scapy/contrib/bgp.py

```diff
@@ -8,162 +6,621 @@
 from scapy.packet import *
 from scapy.fields import *
 from scapy.layers.inet import TCP
+from scapy.contrib.bgp_fields import *
+
+def Class_Dispatcher(s,class_dict,def_class,index_from=None):
```

Comment:
This name is invalid for a function. Please call it something like `class_dispatcher` or better `_class_dispatcher` (since it is an internal function).